src/types.ts

```diff
@@ -125,6 +126,8 @@ export interface AdxTableSchema {
 export interface AdxColumnSchema {
   Name: string;
   CslType: string;
+  Type?: string;
```

Comment:
can you talk about these changes a little more? I'm not sure I see where they are being used in this pr

Reply:
not needed (yet), I forgot to remove this in this PR.